package.json

```diff
@@ -5,31 +5,22 @@
   "license": "Apache-2.0",
   "author": "Google Inc.",
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=8"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
   },
-  "main": "src/index.js",
   "private": true,
   "scripts": {
-    "lint": "eslint '**/*.js'",
-    "pretest": "npm run lint",
-    "unit-cover": "nyc --cache npm test && nyc report --reporter=html",
-    "system-cover": "nyc --cache npm run system-test && nyc report --reporter=html",
-    "test": "npm run unit-test && npm run system-test",
```

Comment:
Just an idea - but for contributors, would it be nice if we had `test` do a `echo tests must be run in the sample directory` or something like that?

Reply:
Good idea! Easier than finding that in the docs. Will update.